rdm/init_files/documents/risk_management_plan.md

```diff
@@ -0,0 +1,74 @@
+---
+id: RMP-001
+revision: 1
+title: Risk Management Plan
+---
+
+# Purpose
+
+This risk management plan describes the activites that are performed in order to
```

Comment:
I think this Risk Management Plan conflicts a bit with the risk-related activities in the Software Plan. E.g., see the planning activity, risk assessment, and risk control activities. Note that the current software plan's approach isn't very good, so I'm not proposing that we keep it, but perhaps we should add a TODO: comment at the top of this file that notes that this document conflicts with the Software Plan.